cmake_options.cmake

```diff
@@ -12,8 +12,8 @@ option(TESTS_NODATA "Build tests for no-data testing")
 set(FAILED_CHECK_ACTION "IGNORE" CACHE STRING "What action to perform on a failed RW_CHECK (in debug mode)")
 set_property(CACHE FAILED_CHECK_ACTION PROPERTY STRINGS "IGNORE" "ABORT" "BREAKPOINT")
 
-set(FILESYSTEM_LIBRARY "BOOST" CACHE STRING "Which filesystem library to use")
-set_property(CACHE FILESYSTEM_LIBRARY PROPERTY STRINGS "CXX17" "CXXTS" "BOOST")
+set(FILESYSTEM_LIBRARY "CXX17" CACHE STRING "Which filesystem library to use")
```

Comment:
This option can go completely as `CXXTS` does not make sense in c++17.

Reply:
It seems my comments were incorrect. Apple has not yet a `<filesystem>` library. It does have an experimental filesystem library though..